codalab-service.py

```diff
@@ -0,0 +1,387 @@
+#! /usr/bin/python2.7
+
+import argparse
+import errno
+import os
+import subprocess
+import test_cli
+
+
+def main():
+    args = CodalabArgs.get_args()
+    service_manager = CodalabServiceManager(args)
+    service_manager.execute()
+
+
+class CodalabArgs(argparse.Namespace):
+    DEFAULT_ARGS = {
+        'version': 'latest',
+        'dev': False,
+        'push': False,
+        'docker_user': None,
+        'docker_pwd': None,
+        'build_locally': False,
+        'image': 'all',
+        'test_build': False,
+        'user_compose_file': None,
+        'start_worker': False,
+        'initial_config': False,
+        'root_dir': None,
+        'instance_name': 'codalab',
+        'mysql_root_pwd': 'mysql_root_pwd',
```

Comment:
see comments above about using `password` and consistency on passwords

Reply:
done (changed to `codalab`)